8/run-client-docker.sh

```diff
@@ -4,8 +4,8 @@
 # Code below gets local IP for dockerized applications to function and communicate correctly.
 ADVERTISEDIP=`ifconfig | grep -Eo 'inet (addr:)?([0-9]*\.){3}[0-9]*' | grep -Eo '([0-9]*\.){3}[0-9]*' | grep -v -m1 '127.0.0.1'`
 GATEWAYPORT=3001
-MEMBERSHIPTABLE="OrleansMembership"
-AWSREGION="us-west-2"
+MEMBERSHIPTABLE=OrleansMembership
+AWSREGION=http://$ADVERTISEDIP:8042
```

Comment:
? 🤔

Reply:
There's a rationale here, I can add it as a comment instead. It'll allow you to point at a local dynamo db if you want. :D